RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptionsViewModel.swift

```diff
@@ -26,6 +26,7 @@ import RevenueCat
 class ManageSubscriptionsViewModel: ObservableObject {
 
     let screen: CustomerCenterConfigData.Screen
+    let appearance: CustomerCenterConfigData.Appearance
```

Comment:
I'm curious what the ViewModel is going to use the `Appearance` for? At first glance it seems like a strictly UI concern (after it is initially fetched).

Reply:
so it probably should live in the View right?

Reply:
Yeah that would be my intuition. If the ViewModel were the one fetching and providing it to the View, you could argue it's "data", but I don't think that's what's happening here?